### ISSUE TITLE
Warnings

### ISSUE THREAD

**Issue by NikitaAtNimble** [galavebalaji/travelbooks#28]
This pull request - https://github.com/galavebalaji/travelbooks/pull/13/files saying that SwiftLint issues resolved, but there is still some of them in project.

**Comment by galavebalaji** [comment]
Yes. There are few warnings has been left because I don't see them that much important at this scope. 
for example:
```
var error: Error?
guard error != nil  else {
completion(.failure(error!))
}
```
